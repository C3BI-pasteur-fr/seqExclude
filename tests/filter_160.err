file test.fa: Excluded 2 from 4 (50.000000 %)
