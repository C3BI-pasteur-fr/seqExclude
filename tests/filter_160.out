>seq1
aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa
tttttttttttttttttttttttttttttttttttttttttttttttttttttttttttttttttttttttttttttttt
cccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccc
gggggggggggggggggggggggggggggggggggggggggggggggggggggggggggggggggggggggggggggggg
>seq4
aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa
tttttttttttttttttttttttttttttttttttttttttttttttttttttttttttttttttttttttttttttttt
cccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccc
gggggggggggggggggggggggggggggggggggggggggggggggggggggggggggggggggggggggggggggggg
